samples/exampleMainRoomList.csv

```diff
@@ -0,0 +1,239 @@
+Floor,Room,Room Type,Connected To Room,First Name,Last Name,Check-in Date,Check-out Date,Art Room,Art Name / Placed Name,Placed By,Changeable,Change Reason,Notes,Paying guest?,Ticket ID in SecretParty,Placement Team Notes,,ROOMBAHT

```

Comment:
Flagging for future us that we should have some sample data that includes more fields - at least fields that we plan on bubbling up in the gooey.

Reply:
sounds good! 
ja the dataset i had contains peoples names and emails, from last year. so I filtered for only the rooms marked for roombot before adding them to the csv(which is why they all end in `,R`). Would be good to get a copy of the complete spreadsheet from this yr.

Reply:
I'm behind on my task of updating the schema, that will help clarify